frontend/public/tutorial.md

```diff
@@ -1,12 +1,64 @@
 ## Getting Started
 
-- In `main.tsx`, update the options passed `SquidContextProvider` with you `appId` and `squidDeveloperId`.
-- Run the `Create .env file` script in (found in the [Squid Console](https://console.squid.cloud)) in the `backend` directory.
-- Run `squid start` in the backend directory.
+Welcome to your first Squid project! Through a few simple steps, you will add functionality to a full-stack to-do application and learn about some of the features of Squid.
 
-## Creating a Todo
+1. In the [Squid Cloud Console](https://console.squid.cloud) create a new app called `todo`.
 
-Querying for todos. In `App.tsx` add the following:
+2. To initialize Squid in a client app, you must provide some details about your Squid application. In the `main.tsx` file of the frontend, update the options passed to `SquidContextProvider` with your `appId` and `squidDeveloperId`. You can find these values in the [Squid Cloud Console](https://console.squid.cloud):
```

Comment:
I'd maybe specify "You can find these values in the **application overview page** of the Squid Cloud Console"